articles/multifactor-authentication/custom.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Configuring Custom Multi-factor Authentication
+url: /multifactor-authentication/custom
+description: Examples for configuring custom MFA implementations.
+topics:
+  - mfa
+  - custom-mfa
+contentType:
+  - how-to
+  - concept
+useCase:
+  - customize-mfa
+---
+# Configuring Custom MFA
+
+You may configure a [rule](/rules) from your [Dashboard > Rules](${manage_url}/#/rules) for custom MFA processes, which allow you to define the conditions that will trigger additional authentication challenges, such as changes in geographic location or logins from unrecognized devices. You can also use the rule to force a particular MFA provider despite the Dashboard toggles, or to force MFA only for particular applications or for particular users.
+
+## The provider setting
+
+The `provider` setting now defaults to `any`. This allows the toggles on the [Dashboard > MFA](${manage_url}/#/mfa) to control which providers are enabled, and the user to select from any of the enabled providers. Setting `provider` to `guardian`, for example, will now restrict end users to only use Guardian for their second factor, regardless of which factors are enabled in the Dashboard. In most cases, it would be recommend to leave the setting as is, rather than modifying it with a rule, and instead control the available factors via the Dashboard toggles.
+
+In addition to `any` or a particular value such as `guardian`, `sms`, `google-authenticator`, or `duo`, you can also set `provider` to `none`. This will forcibly remove MFA prompts from the login process.
+
```

Comment:
'none' is not supported yet, right @santiagoaguiar ?

Reply:
It is not, added the story on the dashboard epic.

Reply:
I'll remove for now